packages/odyssey-storybook/.storybook/OdysseyTheme.js

```diff
@@ -0,0 +1,39 @@
+import { create } from "@storybook/theming";
+
+export default create({
+  base: "light",
+
+  colorPrimary: "#1662dd",
+  colorSecondary: "#00297a",
+
+  // UI
+  appBg: "#ffffff",
+  appContentBg: "#ffffff",
+  appBorderColor: "#d7d7dc",
+  appBorderRadius: 4,
+
+  // Typography
+  fontBase:
+    '"Public Sans", "-apple-system", "BlinkMacSystemFont", "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", "Noto Sans Arabic", sans-serif',
+  fontCode:
+    '"Inconsolata", "SFMono-Regular", "Consolas", "Liberation Mono", "Menlo", "Courier", monospace',
+
+  // Text colors
+  textColor: "#1d1d21",
+  textInverseColor: "#ffffff",
+
+  // Toolbar default and active colors
+  barTextColor: "#1662dd",
+  barSelectedColor: "#1d1d21",
+  barBg: "#f5f5f6",
+
+  // Form colors
+  inputBg: "#ffffff",
+  inputBorder: "#8c8c96",
+  inputTextColor: "#1d1d21",
+  inputBorderRadius: 4,
+
+  brandTitle: "Odyssey Design System",
+  //brandUrl: 'https://odyssey.okta.com',
+  //brandImage: 'https://place-hold.it/350x150',
```

Comment:
curious - could you build this using entirely content exported from `@okta/odyssey-design-tokens` at this point? What would we be missing?

Reply:
Yup! Everything here uses existing tokens straight from the palette, so we could use those here if SB were set up to ingest them.

Reply:
@edburyenegren-okta that is great - but I see the theme values hardcoded here, I was expecting to see something like this:

```
import { colors, font, ... } from "@okta/odyssey-design-tokens"
```

where we compose the values here

Reply:
Im going to take on this chunk of work, lets not let it hold up the merge. I will handle in a FF.

Reply:
@arnoldsandoval-okta and I fixed up the tokens package to get it working locally; however, I'm now getting [this error in CI](https://github.com/okta/odyssey/runs/4218311893?check_suite_focus=true): `ERR! Module not found: Error: Can't resolve '@okta/odyssey-design-tokens' in '/home/runner/work/odyssey/odyssey/packages/odyssey-storybook/.storybook'`

Looks like our tokens package isn't prepped for consumption correctly. You'll also note that `yarn.lock` wasn't updated despite the new dependency in `odyssey-storybook`, which feels like a smell.

Reply:
Adding the package via `yarn add` fails, which is why `yarn.lock` hasn't been updated yet either.

Reply:
Setting this down for the night, but confirming that `yarn build-storybook` also fails locally while relying on `@okta/odyssey-design-tokens`. Local development is still 💯.